zuul.d/project-templates.yaml

```diff
@@ -102,23 +90,12 @@
         - ansible-test-cloud-integration-aws-py36:
             voting: false
     gate:
-      jobs:
-        - ansible-test-sanity-docker
-        - ansible-test-sanity-docker-stable-2.9
-        - ansible-test-sanity-docker-stable-2.10
-        - ansible-test-units-community-aws-python38
-        - build-ansible-collection:
-            required-projects:
-              - name: github.com/ansible-collections/amazon.aws
-              - name: github.com/ansible-collections/ansible.utils
-              - name: github.com/ansible-collections/community.general
-        - ansible-test-cloud-integration-aws-py36:
-            voting: false
+      jobs: *ansible-collections-community-aws-jobs
```

Comment:
This isn't right, we shouldn't have non-voting jobs in gate.  The fix here, is to remove that job, over anchor